CMSgov/easi-app
# [EASI-2570] Remove IT Navigator help text from bottom of new request page

# EASI-2570

## Changes and Description

- Remove IT Navigator help text when selecting a new request's type.

This does _not_ address all of the changes noted in EASI-2570. The final bullet in that ticket (to remove this IT Navigator Help text) was prioritized to be done ahead of the rest of the work in that ticket.

## How to test this change

- `scripts/dev up`
- Login
- Create a new request
- Ensure there's not any help text!

## Screenshots

### Before
![image](https://user-images.githubusercontent.com/15203744/218487176-859c11f9-5db6-41cf-9844-3c09ad1aa692.png)

### After
![image](https://user-images.githubusercontent.com/15203744/218487204-86c8cd58-42d5-4997-b67d-a783929b439a.png)


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
